index.js

```diff
@@ -8,19 +8,27 @@ const fetch = require('./fetch')
 const debug = createDebug('iheart')
 
 // go through a CORS reverse proxy so that these APIs work in the web browser
-const corsProxy = 'https://cors.now.sh/'
+const corsDefaultProxy = 'https://cors.now.sh/'
 
 const searchBase  = 'https://api.iheart.com/api/v1/catalog/searchAll'
 const streamsBase = 'https://api.iheart.com/api/v2/content/liveStations/'
 
+function getProxyURL(options) {
+  return options && 'string' === typeof options.proxy
```

Comment:
With the suggestion above, we can refactor this to:

```js
  return options && typeof options.proxy === 'string' ? options.proxy : corsDefaultProxy
```

Reply:
This may work out unless someone (er... I cannot think of any real world case, but anyway) wants to disable proxy while being in a browser. Of course, `''` can be specified, but it is not as beautiful as `null` (or `false`)

Reply:
Ya `''` is what I was thinking. I'm down for `null` and/or `false` support though.

Reply:
This is what I'm saying. If we simplify the condition like above, good bye `null` and `false`. So what exactly would you like to see? Keep it as is or simplify?

Reply:
Let's leave it as is for now. I think your original version didn't support false though. Up to you if you want to fix that.

Reply:
Oh nvm, I see the boolean check now. Ok so yours is perfect 👍

Reply:
This is funny, because I do not see a boolean check :)